### PR TITLE
Rhys/new tracking implementation

### PR DESCRIPTION
**Pull request by wheresrhys**
## Describe your changes
Adds o-tracking to the astro site and the new o4e site

### CODE REVIEW

File: apps/for-everyone-website/astro.config.mjs [hunk 1]
```diff
@@ -40,6 +40,53 @@ export default defineConfig({
 			social: {
 				github: 'https://github.com/financial-times/origami/',
 			},
+			head: [
+		    // Example: add Fathom analytics script tag.
```
Comment (notlee) [inline apps/for-everyone-website/astro.config.mjs:44]:
```suggestion
```

File: apps/astro-website/public/tracking.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const oTracking = window.Origami['o-tracking'];
```
Comment (notlee) [inline apps/astro-website/public/tracking.js:1]: Do we need to include this in `apps/astro-website/src/layouts/Head.astro`, or is there some auto-astro-magic here?

Comment (wheresrhys) [reply apps/astro-website/public/tracking.js:1]: What do you mean by 'this' in this sentence?

Comment (notlee) [reply apps/astro-website/public/tracking.js:1]: Oh, I mean the script itself `apps/astro-website/public/tracking.js`, I can't see how it's included in the html?

Comment (wheresrhys) [reply apps/astro-website/public/tracking.js]: fixed now

**Review by notlee — APPROVED** [review]

**Pull request merged**